frontend/packages/kubevirt-plugin/integration-tests/tests/vm.tab.dashboard.scenario.ts

```diff
@@ -54,46 +53,101 @@ describe('Test VM dashboard', () => {
   });
 
   it('ID(CNV-3333) Inventory card', async () => {
-    expect(vmInventoryNICs.getText()).toEqual('1 NIC');
-    expect(vmInventoryNICs.$('a').getAttribute('href')).toMatch(
+    expect(dashboardView.vmInventoryNICs.getText()).toEqual('1 NIC');
+    expect(dashboardView.vmInventoryNICs.$('a').getAttribute('href')).toMatch(
       new RegExp(`.*/k8s/ns/${vm.namespace}/${VirtualMachineModel.plural}/${vm.name}/nics`),
     );
-    expect(vmInventoryDisks.getText()).toEqual('2 Disks');
-    expect(vmInventoryDisks.$('a').getAttribute('href')).toMatch(
+    expect(dashboardView.vmInventoryDisks.getText()).toEqual('2 Disks');
+    expect(dashboardView.vmInventoryDisks.$('a').getAttribute('href')).toMatch(
       new RegExp(`.*/k8s/ns/${vm.namespace}/${VirtualMachineModel.plural}/${vm.name}/disks`),
     );
 
     await vm.addDisk(hddDisk);
     await vm.addNIC(multusNetworkInterface);
     await vm.navigateToTab(TAB.Overview);
 
-    expect(vmInventoryNICs.getText()).toEqual('2 NICs');
-    expect(vmInventoryDisks.getText()).toEqual('3 Disks');
+    expect(dashboardView.vmInventoryNICs.getText()).toEqual('2 NICs');
+    expect(dashboardView.vmInventoryDisks.getText()).toEqual('3 Disks');
 
     await vm.removeDisk(hddDisk.name);
     await vm.removeNIC(multusNetworkInterface.name);
   });
 
-  it('ID(CNV-3330) Status card', async () => {
-    await vm.waitForStatus(VM_STATUS.Off);
-    await vm.navigateToOverview();
-    expect(vmStatus.getText()).toEqual(VM_STATUS.Off);
+  it(
+    'ID(CNV-3330) Status card',
+    async () => {
+      await vm.waitForStatus(VM_STATUS.Off);
+      await vm.navigateToOverview();
+      expect(dashboardView.vmStatus.getText()).toEqual(VM_STATUS.Off);
 
-    await vm.action(VM_ACTION.Start, true, VM_BOOTUP_TIMEOUT_SECS);
-    await vm.navigateToTab(TAB.Overview);
-    expect(vmStatus.getText()).toEqual(VM_STATUS.Running);
-  });
+      await vm.action(VM_ACTION.Start, true, VM_BOOTUP_TIMEOUT_SECS);
+      await vm.navigateToTab(TAB.Overview);
+      expect(dashboardView.vmStatus.getText()).toEqual(VM_STATUS.Running);
+      await browser.wait(until.stalenessOf(dashboardView.vmStatusAlert));
+    },
+    VM_CREATE_AND_EDIT_AND_CLOUDINIT_TIMEOUT_SECS,
+  );
 
   it('ID(CNV-3332) Details card', async () => {
-    expect(vmDetailsName.getText()).toEqual(vm.name);
-    expect(vmDetailsNamespace.getText()).toEqual(vm.namespace);
-    expect(vmDetailsNode.getText()).not.toEqual(NOT_AVAILABLE);
-    expect(vmDetailsIPAddress.getText()).not.toEqual(NOT_AVAILABLE);
+    await browser.wait(waitForStringInElement(dashboardView.vmDetailsHostname, vm.name));
+    expect(dashboardView.vmDetailsName.getText()).toEqual(vm.name);
+    expect(dashboardView.vmDetailsNamespace.getText()).toEqual(vm.namespace);
+    expect(dashboardView.vmDetailsNode.getText()).not.toEqual(NOT_AVAILABLE);
```

Comment:
can we test the hostname as well?

Reply:
the hostname is tested above within `await`, the installation of guest agent takes very long time, the monitor of guest agent is in `Status card`, even the alert message disappeared, the hostname may not shows immediately, hence use `await` for it.

Reply:
ah okay, I am sorry I missed that